src/Goerli-DssSpell.sol

```diff
@@ -255,13 +255,13 @@ contract DssSpellAction is DssAction {
 
         // PSM tin increases
         // Increase PSM-USDP-A tin from 0% to 0.1%
-        DssExecLib.setValue(MCD_PSM_PAX_A, "tin", 1 * WAD / 1000);
+        DssExecLib.setValue(MCD_PSM_PAX_A, "tin", 10 * WAD / 10000);
         // Increase PSM-GUSD-A tin from 0% to 0.1%
-        DssExecLib.setValue(MCD_PSM_GUSD_A, "tin", 1 * WAD / 1000);
+        DssExecLib.setValue(MCD_PSM_GUSD_A, "tin", 10 * WAD / 10000);
 
         // PSM tout decrease
         // Reduce PSM-GUSD-A tout from 0.2% to 0.1%
-        DssExecLib.setValue(MCD_PSM_GUSD_A, "tout", 1 * WAD / 1000);
+        DssExecLib.setValue(MCD_PSM_GUSD_A, "tout", 10 * WAD / 10000);
```

Comment:
Would these be better with a local variable?

```solidity
uint256 zero_one_pct = 10 * WAD / 10000;
DssExecLib.setValue(MCD_PSM_PAX_A, "tin", zero_one_pct);
```

Reply:
Will do this if only to save a little gas.

I'm going to add a card to add a dss-exec-lib function for this, as the calculation is different from other bps conversions in the system.

Reply:
https://github.com/makerdao/spells-goerli/pull/142/commits/2fc4cac01e9803e0d51f7ac22fdfb366730dd479